<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddAlgorithm</class>
 <widget class="QDialog" name="DialogAddAlgorithm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Hozzáad</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Kenszöl</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TextFileName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kiválasztot fájl:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addFileButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fájl hozzáadása</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="TextNameOfAlgo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Algoritmus elnevezése:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
